content/control-api/testing-with-postman.textile

```diff
@@ -0,0 +1,86 @@
+---
+title: Testing with Postman
+meta_description: "Control API provides a REST API to allow you to manage your Ably account."
+meta_keywords: "Ably, account management, control, REST API, Control API."
+section: control-api
+index: 30
+jump_to:
+  Help with:
+    - Importing into Postman#importing
+    - Sending a request#send-request
+    - See also#see-also
+---
+
+There are several ways you can try out the API. You can use a command line tool such as "Curl":https://curl.se/ or "HTTPie":https://httpie.io/ to test out various requests. Another way is to use a graphical tool such as Postman or Paw. This topic looks at testing with Postman.
+
+h2(#importing). Importing the specification into Postman
+
+A convenient way to try out the REST APIs is by importing the OAS3 specification into "Postman":https://www.postman.com/. To do this, carry out the following steps: 
+
+1. Make sure you have "Postman":https://www.postman.com/ installed.
+
+2. Start Postman and select File > Import from the main menu. The import dialog is displayed:
+
+<a href="/images/screenshots/control-api/postman-import.png" target="_blank">
+  <img src="/images/screenshots/control-api/postman-import.png" style="width: 100%" alt="Postman import dialog">
+</a>
+
+3. Drag and drop the OAS3 specification you want to test, or use the "Upload file" button to select and import a specification. Alternatively, if you haven't already downloaded the specification, you can provide a link to it in the Ably Open Specs "GitHub repository":https://github.com/ably/open-specs/definitions/control-v1.yaml.
+
+4. On the Import dialog simply leave the default settings, and click "Import".
+
+5. The imported specification will now appear in your Postman collections:
+
+<a href="/images/screenshots/control-api/imported-collection.png" target="_blank">
+  <img src="/images/screenshots/control-api/imported-collection.png" style="width: 100%" alt="Postman imported collection">
+</a>
+
+You are now ready to test out the API.
+
+h2(#send-request). Sending a request
+
+Now that you have imported the specification into Postman, you are ready to try out the API. For Control API you need to configure your token. This token is used to authenticate your requests.
+
+See the "documentation":#authentication on how to obtain your token.
+
+For many Control API requests you will also need to specify your Account ID or Application ID. You can see the "documentation":#ids for details on how to obtain these.
+
+Now that you understand where to obtain these items, you will see how to configure and run a request in postman. 
+
+1. The following screenshot shows the information for the "List account apps" request:
+
+<a href="/images/screenshots/control-api/app-list-request.png" target="_blank">
+  <img src="/images/screenshots/control-api/app-list-request.png" style="width: 100%" alt="App list request">
+</a>
+
+You need to configure your account ID in this case.
```

Comment:
This is an account-level operation and therefore requires your Account ID in the `account_id` request parameter.